setup.py

```diff
@@ -45,9 +45,10 @@
 ]
 
 gcp_require = [
-    'google-api-python-client>=1.6.1',
-    'google-cloud-storage==0.22.0',
-    'oauth2client>=4.1.2'
+    'google-api-python-client>=1.7.9',
```

Comment:
I am a little cautious about the dependency versioning. We've gotten burned a few times by that.

Reply:
Do you have more details on what went wrong in the past?  I'd be happy to add some test cases to improve confidence -- for the sake of future updates I'd like to make sure we can safely move these forward.

Reply:
I think the main thing is that since cloudaux is a library, we want to avoid having to pin or define version numbers. Namely, if a project makes use of cloudaux, they will be bound to the versions that cloudaux specifically requires. 

We had issues where some other libraries were dependencies that defined their own pinned versions that conflicted with cloudaux and it created some annoying issues that we needed to fix.

Reply:
Makes sense, I've reverted the library changes and verified these new resources work with the older version of the library (after a workaround for a bug in the older libraries).  Please take another look!